build.gradle

```diff
@@ -137,7 +137,7 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.7.4',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:4.1.0.CR6',
+                netty: 'io.netty:netty-codec-http2:[4.1.0.CR6]',
```

Comment:
Yes. Gradle is able to handle the brackets, but doesn't change behavior based on them (tested using Android projects), which is a bit unfortunate. They do change Maven's behavior though (tested using pom.xml from #1415)